For cross environment, it not possible to run the generated executable.
nstead use the build version of libtool to generate the man pages. 

Date: 2010/07/09
Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>

Index: libtool-2.2.10/Makefile.am
===================================================================
--- libtool-2.2.10.orig/Makefile.am
+++ libtool-2.2.10/Makefile.am
@@ -337,7 +337,7 @@ update_mans = \
   PATH=.$(PATH_SEPARATOR)$$PATH; export PATH; \
   $(HELP2MAN) --output=$@
 $(srcdir)/doc/libtool.1: $(srcdir)/$(auxdir)/ltmain.sh
-	$(update_mans) --help-option=--help-all libtool
+	$(update_mans) --help-option=--help-all ${build_alias}-libtool
 $(srcdir)/doc/libtoolize.1: $(srcdir)/libtoolize.in
 	$(update_mans) libtoolize
 
